force-app/main/default/objects/Event_Registration__c/Event_Registration__c.object-meta.xml

```diff
@@ -160,7 +160,10 @@
         <type>AutoNumber</type>
```

Comment:
@bmtemakina I believe this change is related to Event Registration object, is that correct ? if so please remove this metadata form this PR and create another PR for this change.